crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -636,6 +650,21 @@ pub struct RenderMeshInstancesCpu(MainEntityHashMap<RenderMeshInstanceCpu>);
 #[derive(Default, Deref, DerefMut)]
 pub struct RenderMeshInstancesGpu(MainEntityHashMap<RenderMeshInstanceGpu>);
 
+/// The result of an entity removal.
+///
+/// Whenever entity data is removed from the [`MeshInputUniform`] buffer, the
```

Comment:
Is this so that the buffer is always compact so that shaders can dispatch over it without gaps?

Reply:
If so please document this.

Reply:
Yep, you're right. Added some documentation.